### ISSUE TITLE
Increase coding area

### ISSUE THREAD

**Issue by J-Moravec** [ronreiter/interactive-tutorials#815]
On my screen, the coding area can show ~ 6 lines.

Even 1970 [IBM 3277](https://en.wikipedia.org/wiki/IBM_3270) supported double of that (40 x 12).

That makes coding anything but the simplest stuff super annoying.

Dunno why there isn't a toggle to increase the size and the coding area needs to take ~ 20% of the screen.

**Comment by GaryNapier** [comment]
Yes, can I also please add to this. There is just a tiny little box at the bottom and you cannot adjust the size and it's incredibly annoying and very poor design. Can I please request that for coding exersises the screen is split vertically (text describing the task on the left and code on the right) so that you can see everything, also with adjustable panels. At the moment it feels like you're trying to write code through a keyhole. 

<img width="1355" height="768" alt="Image" src="https://github.com/user-attachments/assets/0b599ec9-c7e3-459e-b3a5-e2c350fef72c" />

Something like the above, however the little code box is also restricted and the Output box should be put below the code box, and also made adjustable (sorry, I don't know how to change the rest of these elements).

Many, many many thanks in advance!
